backup-stores/azure/src/main/java/io/camunda/zeebe/backup/azure/FileSetManager.java

```diff
@@ -50,6 +50,23 @@ void save(final BackupIdentifier id, final String fileSetName, final NamedFileSe
     }
   }
 
+  public void delete(final BackupIdentifier id, final String fileSetName) {
+    assureContainerCreated();
+    final ListBlobsOptions options = new ListBlobsOptions().setPrefix(fileSetPath(id, fileSetName));
+    containerClient
+        .listBlobs(options, Duration.of(10, ChronoUnit.SECONDS))
```

Comment:
:question: What happens when we hit the timeout?

Reply:
Since we don't. use the limit duration in the other queries I decided to remove here as well